.circleci/config.yml

```diff
@@ -814,7 +814,8 @@ workflows:
       - playground
```

Comment:
Should this also be filtered?

Reply:
I think you are right, but I'll make a separate PR.